core/src/main/java/org/jdbi/v3/core/extension/ExtensionHandler.java

```diff
@@ -44,6 +45,15 @@ public interface ExtensionHandler {
     /** Handler that only returns null independent of any input parameters. */
     ExtensionHandler NULL_HANDLER = (handleSupplier, target, args) -> null;
 
+    /** Constant for {@link Object#equals(Object)}. */
+    Method EQUALS_METHOD = JdbiClassUtils.methodLookup(Object.class, "equals", Object.class);
```

Comment:
I originally had these in `JdbiClassUtils` in order to have them not be part of the public API. Do we really want to ship this as part of public API?